tests/unit/s2n_aead_aes_test.c

```diff
@@ -343,7 +345,7 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_stuffer_wipe(&conn->in));
 
         /* Tamper with the IV and ensure decryption fails */
-        for (int j = 0; j < S2N_TLS_GCM_IV_LEN; j++) {
+        for (int j = 0; j < S2N_TLS_GCM_EXPLICIT_IV_LEN; j++) {
```

Comment:
nit: use `size_t` instead of `int` here.

Reply:
Changed all counters from `int` to `size_t` and made sure there are not signed/unsigned comparison surprises in the for-loops.